en/docs/setup/removing-unused-tokens-from-the-database.md

```diff
@@ -10,9 +10,46 @@ avoid a growing access token table.
 
 You can use one of the following methods for token cleanup.
 
+!!! note
+    We recommend using stored procedures instead of the WSO2 Identity Server to cleanup tokens since there 
```

Comment:
Could we explain what we mean by `cleaning up tokens using WSO2 Identity Server`?

Reply:
Fixed with https://github.com/wso2/docs-is/pull/2891/commits/94dc4de619234388f1e95f1bbd8a03ec7ab875ea.